src/App.tsx

```diff
@@ -4,6 +4,7 @@ import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
 import ProfilePage from 'src/pages/ProfilePage/ProfilePage';
 import AccountSettingsView from 'src/pages/ProfilePage/views/AccountSettingsView/AccountSettingsView';
 import SellHouseView from 'src/pages/ProfilePage/views/SellHouseView/SellHouseView';
+import MyHousesView from 'src/pages/ProfilePage/views/MyHousesView/MyHousesView';
```

Comment:
Could you run `prettier` in this file? There are a couple of warnings because of the order of imports.